library/dd-sdk-android-logs/src/main/kotlin/com/datadog/android/log/internal/logger/DatadogLogHandler.kt

```diff
@@ -77,7 +75,23 @@ internal class DatadogLogHandler(
         }
 
         if (level >= AndroidLog.ERROR) {
-            GlobalRum.get().addError(message, RumErrorSource.LOGGER, throwable, attributes)
+            val rumFeature = sdkCore.getFeature(Feature.RUM_FEATURE_NAME)
+            if (rumFeature != null) {
+                rumFeature.sendEvent(
+                    mapOf(
+                        "type" to "logger_error",
+                        "message" to message,
+                        "throwable" to throwable,
+                        "attributes" to attributes
+                    )
+                )
+            } else {
+                internalLogger.log(
+                    InternalLogger.Level.INFO,
```

Comment:
Maybe `info` is a bit too much for this. Customers who don't use RUM will get "spammed" if they log many errors, wdyt?

Reply:
Indeed, good point. The problem is a bit wider imo, because maybe we need to notify only once that feature is missing during the SDK instance lifetime.

I created a dedicated task to tune up the logging for SDK v2.